### PR TITLE
Rasterize - using Chrome instead of Phantom JS

### PR DESCRIPTION
**Pull request by orenzohar**
## Status
on hold (waiting for 5.0)

## Related Issues
fixes: https://github.com/demisto/etc/issues/17062
fixes: https://github.com/demisto/etc/issues/17769

## Description
new implementation of Rasterize, using chrome instead of PhantomJS
 
## Related PRs
new image: https://github.com/demisto/dockerfiles/pull/333

## Required version of Demisto
5.0.0

## Does it break backward compatibility?
   - Yes
       - Further details: will use a new image

## Must have
- [x] Tests
- [x] Documentation (with link to it)
- [x] Code Review
- [ ] Check content update over old and new GA

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [CHANGELOG only](https://github.com/demisto/content/blob/84bf79747b0fcd8196ad3359321399d85f5129d4/Integrations/rasterize/CHANGELOG.md)

### CODE REVIEW

File: Integrations/rasterize/rasterize.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ configuration:
 description: Rasterize URLs or emails given to it
 display: Rasterize
 name: Rasterize
+fromversion: "5.0"
```
Comment (yuvalbenshalom) [inline Integrations/rasterize/rasterize.yml]: please change to `fromversion: "5.0.0"`
this what prevents the build from running tests

File: Integrations/rasterize/rasterize.py [hunk 2]
```diff
@@ -1,174 +1,203 @@
...
 
-    with open('htmlBody.html', 'w') as f:
-        f.write('<html style="background:white";>' + html + '</html>')
+    try:
+        demisto.debug('Navigating to url')
 
-    proxy_flag = "--proxy={}".format(HTTP_PROXY) if PROXY else ""
-    demisto.debug('rasterize proxy settings: {}'.format(proxy_flag))
+        driver.get(path)
+        driver.implicitly_wait(5)
```
[the remark this answers is not in the document]
Comment (orenzohar) [reply Integrations/rasterize/rasterize.py]: to load the url/file

File: Integrations/rasterize/rasterize.py [hunk 3]
```diff
@@ -1,174 +1,203 @@
...
+
+def rasterize_command():
     url = demisto.getArg('url')
-    if not (url.startswith("http")):
-        url = "http://" + url
-    friendly_name = 'url.png'
-    if demisto.getArg('type') == 'pdf':
-        friendly_name = 'url.pdf'
+    w = demisto.args().get('width', 600)
+    h = demisto.args().get('height', 800)
```
[the remark this answers is not in the document]
Comment (orenzohar) [reply Integrations/rasterize/rasterize.py]: what do you mean?

File: Integrations/rasterize/rasterize.py [hunk 4]
```diff
@@ -1,174 +1,203 @@
...
+    r_type = demisto.args().get('type', 'png')
+
+    if not (url.startswith('http')):
+        url = f'http://{url}'
```
[the remark this answers is not in the document]
Comment (orenzohar) [reply Integrations/rasterize/rasterize.py]: I agree, kept it to not break BC

File: Integrations/rasterize/rasterize.py [hunk 5]
```diff
@@ -1,174 +1,203 @@
...
+        if r_type.lower() == 'pdf':
+            output = get_pdf(driver, width, height)
+        else:
+            output = get_image(driver, width, height)
+
+        return output
+
+    except NoSuchElementException as ex:
+        if 'invalid argument' in str(ex):
+            return_error(URL_ERROR_MSG) if WITH_ERRORS else return_warning(URL_ERROR_MSG)
```
[the remark this answers is not in the document]
Comment (orenzohar) [reply Integrations/rasterize/rasterize.py]: the exception message, in this case, is not informative. that's why I replaced it. but I'll add it to the end

**Comment by glicht** [comment]
@orenzohar what is the status here? Do we want to merge?

**Comment by orenzohar** [comment]
@glicht yes it's ready

File: Integrations/rasterize/CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+## [Unreleased]
+    - Updated to use chrome driver instead of phantomJS.
```
Comment (glicht) [inline Integrations/rasterize/CHANGELOG.md]:
```suggestion
    - Updated to use chrome driver instead of phantomJS (Requires Demisto 5.0).
```

**Comment by kirbles19** [comment]
@orenzohar I can't access the link to the changelog file, please paste the full URL.

**Pull request merged**